networks/movement/movement-full-node/src/da/stream_blocks/mod.rs

```diff
@@ -38,13 +38,30 @@ impl StreamBlocks {
 				.ok_or(anyhow::anyhow!("No blob type in response"))?
```

Comment:
It's better to handle `None` in a graceful manner here, which would allow future changes in the protocol without having to make a package name break again.